src/rviz/TSRMarker.cpp

```diff
@@ -0,0 +1,14 @@
+#include <aikido/rviz/TSRMarker.hpp>
+
+using dart::dynamics::SimpleFrame;
+
+namespace aikido {
+namespace rviz {
+
+TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> tsrFrames)
```

Comment:
This already doesn't make any copies if the caller uses `std::move`, e.g.:
```c++
// Caller std::move's from local variable to argument:
std::vector<std::unique_ptr<SimpleFrame>> frames = /* ... */;
auto marker = TSRMarker{std::move(frames)};

// Constructor std::moves from argument to member variable:
TSRMarker::TSRMarker(std::vector<std::unique_ptr<SimpleFrame>> frames)
  : markers{std::move(frames)} {}
```
Note that `std::vector<std::unique_ptr<...>>` is not copyable, so we already **must not** be introducing an extra copy. 😉 

My general understanding is that rvalue references are only useful for: (1) move constructors and (2) perfect forwarding in templates due to [reference collapsing](http://stackoverflow.com/a/13726754/111426).